wordlistctl.py

```diff
@@ -245,224 +242,235 @@ def lst_func(parser: argparse.ArgumentParser) -> None:
-				argument["right"] = argument["help"] + " " + ( "(default: '%s')" % argument["default"] if isinstance(argument["default"], str) else "(default: %s)" % str(argument["default"]) )
-			elif ("help" in argument and argument["help"] != "" and not str.isspace(argument["help"])):
-				argument["right"] = argument["help"]
-			elif ("default" in argument and argument["default"] != argparse.SUPPRESS):
-				argument["right"] = "Default: '%s'" % argument["default"] if isinstance(argument["default"], str) else "Default: %s" % str(argument["default"])
-			else:
-				argument["right"] = "No description available"
-			lmaxlen = max(lmaxlen, len(argument["left"]))
-			rmaxlen = max(rmaxlen, len(argument["right"]))
-
-		lwidth = lmaxlen
-		rwidth = max(0, self.width - lwidth - 4)
-		if (lwidth > int(self.width / 2) - 4):
-			lwidth = max(0, int(self.width / 2) - 4)
-			rwidth = int(self.width / 2)
-		outtmp = "  %-" + str(lwidth) + "s  %s"
-
-		lwrapper = textwrap.TextWrapper(width=lwidth)
-		rwrapper = textwrap.TextWrapper(width=rwidth)
-		for argument in self.positionals + self.options:
-			argument["left"] = lwrapper.wrap(argument["left"])
-			argument["right"] = rwrapper.wrap(argument["right"])
-
-
-		if (len(self.positionals) > 0):
-			output.append("")
-			output.append("Positionals:")
-			for positional in self.positionals:
-				for i in range(0, max(len(positional["left"]), len(positional["right"]))):
-					left = positional["left"][i] if (i < len(positional["left"])) else ""
-					right = positional["right"][i] if (i < len(positional["right"])) else ""
-					output.append(outtmp % (left, right))
-
-
-		if (len(self.options) > 0):
-			output.append("")
-			output.append("Options:")
-			for option in self.options:
-				for i in range(0, max(len(option["left"]), len(option["right"]))):
-					left = option["left"][i] if (i < len(option["left"])) else ""
-					right = option["right"][i] if (i < len(option["right"])) else ""
-					output.append(outtmp % (left, right))
-
-		if ("epilog" in self.program and self.program["epilog"] != "" and not str.isspace(self.program["epilog"])):
-			output.append("")
-			output.append(dewrapper.fill(self.program["epilog"]))
-
-		# Return output as single string
-		return str.join("\n", output)
-
-	
-	def print_usage(self, file=None):
-		if (file == None):
-			file = sys.stdout
-		file.write(self.format_usage() + "\n")
-		file.flush()
-
-	
-	def print_help(self, file=None):
-		if (file == None):
-			file = sys.stdout
-		file.write(self.format_help() + "\n")
-		file.flush()
-
-	def error(self, message):
-		sys.stderr.write(self.format_usage() + "\n")
-		sys.stderr.write(("Error: %s" % message) + "\n")
-		sys.exit(2)
+    def __init__(self, *args, width=78,
+                 **kwargs):  # this is basically just to get the variables for the screen, a bit like writing characters to the screen when youre making a kernel, place a character and then increase the y coordinates by x pixles
```

Comment:
I recommend using [docstrings](https://www.python.org/dev/peps/pep-0257/) for comments like this.

Reply:
never